### ISSUE TITLE
Node 14 support - ERR_INVALID_ARG_TYPE error

### ISSUE THREAD

**Issue by Kostanos** [MattiLehtinen/postgrator-cli#21]
The is the error on node 14  (possible also in node 12):

```
# postgrator 6 --host myhost --database mydb --username myuser --password mypass
buffer.js:728
    throw new ERR_INVALID_ARG_TYPE(
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "string" argument must be of type string or an instance of Buffer or ArrayBuffer. Received an instance of Object
    at Function.byteLength (buffer.js:728:11)
    at Writer.addCString (/usr/local/lib/node_modules/pg/node_modules/pg-protocol/dist/buffer-writer.js:41:30)
    at Object.startup (/usr/local/lib/node_modules/pg/node_modules/pg-protocol/dist/serializer.js:9:32)
    at Connection.startup (/usr/local/lib/node_modules/pg/lib/connection.js:124:33)
    at Connection.<anonymous> (/usr/local/lib/node_modules/pg/lib/client.js:121:13)
    at Connection.emit (events.js:315:20)
    at Socket.<anonymous> (/usr/local/lib/node_modules/pg/lib/connection.js:45:12)
    at Object.onceWrapper (events.js:421:28)
    at Socket.emit (events.js:327:22)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1132:10) {
  code: 'ERR_INVALID_ARG_TYPE'
}

# nodejs -v
v14.15.0
```

Possible issuer related to the one mentioned here: https://stackoverflow.com/a/62606989/2215679

**Comment by djmitchella** [comment]
I hit the same thing -- explicitly using `pg@8.2.1` fixed it for me.

**Comment by perrin4869** [comment]
In case you are interested, this is fixed in my fork [`perrin4869/postgrator-cli`](https://www.npmjs.com/package/@perrin4869/postgrator-cli).

**Comment by perrin4869** [comment]
Fixed in #30
